### PR TITLE
LPS-132489 Missing text formating container after double clicking linked blog content 

### PR DESCRIPTION
**Pull request by jgarcialr**
Hello FEI team:

**Description of the issue:**

When you add a link inside the content of a blog and you try to format this content, the text formatting toolbar is not shown.

**Test plan:**
```

1. Start a bundle from master version
2. Go to DM > Add an image
3. Go to blogs > Attempt to add a new blog
4. Type title and content
5. Select all the content > Add a link using added image
6. Double clicking linked blog content
```

**Expected Result:**
Text formating container display. [Expected.mp4](https://issues.liferay.com/secure/attachment/406391/406391_Expected.mp4)

**Current Result:**
Text formating container is missing. [Reproduced.mp4](https://issues.liferay.com/secure/attachment/406392/406392_Reproduced.mp4) 

**Things i've tried:**
Comparing the behavior in other parts of the portal (Message Boards, web content, wiki) but they aren't using alloyeditor by default.
Looking behavior in previous versions without updated ckeditor (7.2)

**Solution:**
After adding autolink to removePlugins the behavior is the same as in previous versions.

### CODE REVIEW

File: modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/editor/configuration/BaseAlloyEditorConfigContributor.java [hunk 1]
```diff
@@ -63,17 +63,16 @@ public void populateConfigJSONObject(
...
 		).put(
 			"imageScaleResize", "scale"
 		).put(
 			"language",
 			StringUtil.replace(getLanguageId(themeDisplay), "iw_", "he_")
 		).put(
 			"removePlugins",
-			"contextmenu,elementspath,floatingspace,image2,link,liststyle," +
-				"resize,table,tabletools,toolbar"
+			"autolink,contextmenu,elementspath,floatingspace,image2,link," +
```
Comment (diegonvs) [inline modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/editor/configuration/BaseAlloyEditorConfigContributor.java:74]: Downloading and running this fix it looks fine to get back the "native" alloy-editor linking panel but I have one thing to be aware: this ck-editor autolink plugin was added there for preventing a performance issue [as this commit explain](https://github.com/liferay-frontend/liferay-portal/commit/11dfcd178398dc0798b756a4f4a141352397bec8)

[This ticket](https://issues.liferay.com/browse/LPS-119875) explain more about this performance issue but testing locally with this current fix it looks already solved. See the following video:


https://user-images.githubusercontent.com/7663513/120027417-b91dc600-bfc9-11eb-9301-e2fac70e2d0a.mov



Hey @john-co or @nkkwon, could someone just see from a QA point of view here?

**Comment by john-co** [comment]
Just started reviewing :)

File: modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/editor/configuration/BaseAlloyEditorConfigContributor.java (hunk 1 above)
Comment (john-co) [reply modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/editor/configuration/BaseAlloyEditorConfigContributor.java:74]: Hi @diegonvs , I took a look and the performance issue looks resolve in this PR. Looks ok from QA standpoint.

https://user-images.githubusercontent.com/35235266/120045620-47437d80-bfc5-11eb-9ff2-a014af46deba.mp4

**Comment by julien** [comment]
ci:forward

**Pull request closed without merging**